lib/kim/Install.py

```diff
@@ -205,37 +187,36 @@ def error():
   except subprocess.CalledProcessError as e:
 
     # Error: but first, check to see if it needs a driver
-
-    firstRunOutput = e.output
+    firstRunOutput = e.output.decode()
 
     cmd = "cd %s/%s; make kim-item-type" % (thisdir,addmodelname)
     txt = subprocess.check_output(cmd,stderr=subprocess.STDOUT,shell=True)
+    txt = txt.decode().strip()
     if txt == "ParameterizedModel":
 
       # Get and install driver
 
       cmd = "cd %s/%s; make model-driver-name" % (thisdir,addmodelname)
       txt = subprocess.check_output(cmd,stderr=subprocess.STDOUT,shell=True)
-      adddrivername = txt
-      print("First Installing model driver: %s" % adddrivername)
+      adddrivername = txt.decode().strip()
+      print("First installing model driver: %s" % adddrivername)
       cmd = "cd %s; python Install.py -a %s" % (thisdir,adddrivername)
       try:
         txt = subprocess.check_output(cmd,stderr=subprocess.STDOUT,shell=True)
       except subprocess.CalledProcessError as e:
         print(e.output)
         sys.exit()
 
-      print(txt)
```

Comment:
This should be put back in.  It echos the output associated with the build and install of the Model Driver.